apps/src/tutorialExplorer/tutorialExplorer.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Entry point to build a bundle containing a set of globals used when displaying
+ * tutorialExplorer.
+ */
+
+require('babel-polyfill');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Immutable from 'immutable';
+import update from 'react-addons-update';
+import FilterSet from './filterSet';
+import TutorialSet from './tutorialSet';
+
+window.TutorialExplorerManager = function (options) {
+  this.options = options;
+
+  const TutorialExplorer = React.createClass({
```

Comment:
I would have TutorialExplorer live outside of TutorialExplorerManager (such that we could test it independently if we so desired)